insights/parsers/sctp.py

```diff
@@ -9,6 +9,9 @@
 
 SCTPAsc - file ``/proc/net/sctp/assocs``
 ----------------------------------------
+
+SCTPSnmp - file ``/proc/net/sctp/snmp``
+----------------------------------------
```

Comment:
nitpick: please remove one `-`, to keep it in the same length with the title.

Reply:
Done